apps/ai-agent/.env.example

```
@@ -0,0 +1,5 @@
+LLM_PROVIDER=gemini
+LLM_API_KEY=
+LLM_MODEL=gemini-3.1-flash
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_MODEL key should go before the LLM_PROVIDER key

(UnorderedKey)